inst/extdata/correctionsataglance-example.json

```diff
@@ -103,267 +103,187 @@
     "type": "Discharge",
     "points": []
   },
-  "fieldVisitMeasurements": [
+  "corrections": [],
+  "reportMetadata": {
+    "country": "UNITED STATES OF AMERICA",
+    "altitude": "  657.64",
+    "requestingUser": "thongsav",
+    "agency": "USGS ",
+    "endDate": "2015-12-24T00:00:00-06:00",
+    "drainageArea": "    2840",
+    "requestDateTime": "2015-12-28T10:26:27.915-06:00",
+    "nwisRaAuthToken": "565e39ff-8e34-4e98-8170-6a3d6fe8b083",
+    "latitude": "37.9299166666667",
+    "county": "PHELPS",
+    "description": "Shows overview of correction information",
+    "title": "Corrections At a Glance",
+    "longitute": "-91.9773333333333",
+    "primaryParameter": "Discharge.ft^3/s@06933500",
+    "primaryTsIdentifier": "0f20e9ce98c646bb9c0d1439d184423d",
+    "requestId": "7450194a-69bd-411d-b0d6-6bbc46dc159d",
+    "contributingDrainageArea": "        ",
+    "altitudeDatumCode": "NGVD29    ",
+    "siteNumber": "06933500       ",
+    "stationName": "Gasconade River at Jerome, MO",
+    "coordinateDatumCode": "NAD83     ",
+    "state": "MISSOURI",
+    "startDate": "2014-12-24T00:00:00-06:00",
+    "stationId": "06933500"
+  },
+  "fieldVisits": [
     {
-      "shiftInFeet": 0.1751196841701,
-      "errorMinShiftInFeet": -0.527682877341,
-      "errorMaxShiftInFeet": 0.8548049457021,
+      "locationIdentifier": "06933500",
+      "startTime": "2015-06-20T12:00:00.000-05:00",
+      "endTime": "2015-06-20T12:30:08.000-05:00",
       "identifier": "25AB4D37DA6E4F73E0534CEB3D98B6F9",
-      "measurementStartDate": "2015-06-20T12:05:49.000-05:00",
-      "ratingModelIdentifier": "Gage height-Discharge.STGQ@06933500",
-      "discharge": 29000,
-      "dischargeUnits": "ft^3/s",
-      "errorMinDischarge": 26680.0000,
-      "errorMaxDischarge": 31320.0000,
-      "measurementNumber": "959",
-      "qualityRating": "FAIR",
-      "historic": false,
-      "meanGageHeight": 14.45,
-      "meanGageHeightUnits": "ft",
-      "shiftNumber": 0
+      "isValid": true,
+      "lastModified": "2015-11-29T03:13:14.845-06:00",
+      "party": "tk,cp",
+      "remarks": "loop rating study qm.",
```

Comment:
No, we are no longer concerned with readings in this data, they are Field Visit level remarks.